README.md

```diff
@@ -1,13 +1,3 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This repo contains Teanna Barrett's portfolio and STEP projects. This should be a creative outlet.
```

Comment:
minor nit:
My TL always says to never say "should" in code or documentation. It either is or isn't (or must or must not).

Reply:
That is a good rule and an easy fix, Thank you!